docs/integration_resources/resources/usd_tutorials/usd_tutorial_01.md

```diff
@@ -0,0 +1,60 @@
+## Creating Your First USD Stage
+
+This tutorial is focused on creating, using the operators from Shift_USD catalog, a simple USD stage containing a transform and a sphere.
+
+In order to follow easily the steps, it is recommended to read the documentation about the [Shift's interface](../../../getting_started/basics/ui_overview.md) and [workflow execution](../../../getting_started/basics/execute.md).
+
+
+### Tutorial
+
+1. Open Shift application. A new window's instance of Shift should appear like the following one:
+
+<figure>
+    <img src="images/usd_tutorial_01/step_01_t01.png" alt="Shift UI.">
+    <figcaption><b>Figure 1</b>: Shift UI.</figcaption>
+</figure>
+
+2. Now, add a new **<tt>UsdStageNew</tt>** node to the board. Using the **Inspector** widget, set as value for the **filepath** input plug a filepath pointing to a `.usda` file. For doing this, the UI offers a browse button next to the plug input field that opens a browse dialog to facilitate locating the file or it can be added typing manually the path. For the current example, the filepath can be: `D:/USD_TUTORIALS/01/HelloWorld.usda`. The board should look like the following one:
```

Comment:
We don't need to be as detailed about how to set values. 
Also, will file paths be relative to the shift folder? 

```suggestion
2. Now, add a new **<tt>UsdStageNew</tt>** node to the board. Using the **Inspector** widget, set as the value for the **filepath** input plug a filepath pointing to a `.usda` file. For the current example, the filepath can be: `D:/USD_TUTORIALS/01/HelloWorld.usda`. The board should look like the following:
```

Reply:
Done. I was working with absolute paths just to avoid confusing the user, because the files are reused in the tutorials and in this way they can see that is the file related to the current tutorial 🤔